diff --git a/Cargo.toml b/Cargo.toml
index 9c51388aa..c44ca66aa 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -48,6 +48,7 @@ uv-keyring = { version = "0.0.28", path = "crates/uv-keyring" }
 uv-logging = { version = "0.0.28", path = "crates/uv-logging" }
 uv-macros = { version = "0.0.28", path = "crates/uv-macros" }
 uv-metadata = { version = "0.0.28", path = "crates/uv-metadata" }
+uv-nix = { version = "0.0.1", path = "crates/uv-nix" }
 uv-normalize = { version = "0.0.28", path = "crates/uv-normalize" }
 uv-once-map = { version = "0.0.28", path = "crates/uv-once-map" }
 uv-options-metadata = { version = "0.0.28", path = "crates/uv-options-metadata" }
